### ISSUE TITLE
Menu overlays content

### ISSUE THREAD

**Issue by SunboX** [hegemonic/jsdoc-baseline#163]
![menu](https://cloud.githubusercontent.com/assets/83344/7854928/ec737918-051d-11e5-96df-e8ec510a808b.PNG)

**Comment by hegemonic** [comment]
Yeah, this is a bug. Ideally we would increase the width of the navigation column when necessary. This might be difficult to accomplish with the tree widget we're currently using, though.

**Comment by SunboX** [comment]
Maybe you can add ellipsis to the long strings, like the MDN docs do it. For example here:
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

**Comment by hegemonic** [comment]
@SunboX Good suggestion. At the very least, that might be a good short-term solution.

**Comment by ProLoser** [comment]
You can add an ellipsis using pure CSS.
